README.md

```diff
@@ -383,6 +411,19 @@ This cost estimate is based on the `us-east-1` region pricing and does not consi
 
 ## Limitations and workarounds
 
+#### WORKAROUND: Patch fetch behaviour for ISR. Only for next@13.5.1+
+
+In order to make fetch work as expected with ISR, you need to patch the `fetch` function in your app. Just add this lines to your root layout component:
+
+```ts
+const asyncStorage = require("next/dist/client/components/static-generation-async-storage.external");
```

Comment:
We should include the function here to make it clear that this code should be inside the function, not at the top of the file.